helm/fhir_service/install_modules.json

```diff
@@ -0,0 +1,13 @@
+[
+    "openid-client",
+    "qewd-template",
+    "moment-timezone",
+    "jsonschema",
+    "valid-url",
+    "bluebird",
+    "fast-safe-stringify",
+    "qewd-router",
+    "qewd-transform-json",
+    "request",
+    "winston"
+]
```

Comment:
"qewd-router", "qewd-transform-json", "request" are not needed here because they are loaded by  qewd-up. See Rob's note: https://github.com/QEWD-Courier/QEWD-Courier/pull/36#pullrequestreview-212769155